332-ci-manager/service/src/main/java/io/harness/ci/execution/validation/CIMiningPatternJob.java

```
@@ -129,6 +148,32 @@ private Set<String> initializeValidDomains() {
     return validDomains;
   }
 
+  private Set<String> initializeWhiteListedDomains() {
+    Set<String> whiteListed = new HashSet<>();
+
+    try {
+      byte[] fileContent = downloadFromGCS(WHITE_LISTED_DOMAINS_FILE);
+
+      String[] patterns = new String(fileContent, StandardCharsets.UTF_8).split("\n");
+
+      if (patterns == null || patterns.length != 0) {
+        whiteListed.clear();
+      }
+
+      for (String pattern : patterns) {
+        if (isNotEmpty(pattern)) {
+          whiteListed.add(pattern.trim());
+        }
+      }
+
+    } catch (Exception e) {
+      log.error("Error initialize white listed domains set for CI hosted");
```

are you ignoring this exception ?

it will log both error and success in case it comes here.

---

ignoring yes. The set will be empty

---

But it will also log success message which is in next line.

---

add return statement.